### PR TITLE
feat: Adds support for `ldcli config --list`

### PR DESCRIPTION
**Pull request by dbolson**
Running `ldcli config --list` will read the values set in the config file. If the file doesn't exist, it's a no-op.

**Requirements**

- [ ] I have added test coverage for new or changed functionality
- [ ] I have followed the repository's [pull request submission guidelines](../blob/main/CONTRIBUTING.md#submitting-pull-requests)
- [ ] I have validated my changes against all supported platform versions

**Related issues**

Provide links to any issues in this repository or elsewhere relating to this pull request.

**Describe the solution you've provided**

Provide a clear and concise description of what you expect to happen.

**Describe alternatives you've considered**

Provide a clear and concise description of any alternative solutions or features you've considered.

**Additional context**

Add any other context about the pull request here.

### CODE REVIEW

File: cmd/cmdtest.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ func CallCmd(
 		membersClient,
 		projectsClient,
 		"test",
+		false,
```
Comment (sunnyguduru) [inline cmd/cmdtest.go:35]: I don't see NewRootCommand modified to have 7 arguments anywhere

**Review by sunnyguduru — COMMENTED** [review]
Changes look alright. I just left a comment about the additional argument being passed into `NewRootCommand`

File: cmd/cmdtest.go (hunk 1 above)
Comment (dbolson) [reply cmd/cmdtest.go:35]: That should [be here](https://github.com/launchdarkly/ldcli/pull/176/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR33).

**Review by sunnyguduru — APPROVED** [review]

**Review by k3llymariee — APPROVED** [review]

File: internal/config/config.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+func GetConfigPath() string {
+	configPath := os.Getenv("XDG_CONFIG_HOME")
+	if configPath == "" {
+		home, err := homedir.Dir()
+		if err != nil {
+			return ""
+		}
+		configPath = filepath.Join(home, ".config")
+	}
```
[the remark this answers is not in the document]
Comment (dbolson) [reply internal/config/config.go:45]: When I tried that I get `/Users/danny/Library/Preferences` locally, which doesn't seem great. It looks like stripe does the same thing we do here, which seems safe enough. We can test the linux and windows binaries and update if it doesn't work.

**Pull request merged**